lightning/src/ln/channel.rs

```diff
@@ -843,16 +846,24 @@ impl<Signer: Sign> Channel<Signer> {
 	}
 
 	/// Returns a minimum channel reserve value the remote needs to maintain,
-	/// required by us.
+	/// required by us according to the configured or default
+	/// [`ChannelHandshakeConfig::their_channel_reserve_proportion_millionth`]
 	///
 	/// Guaranteed to return a value no larger than channel_value_satoshis
 	///
-	/// This is used both for new channels and to figure out what reserve value we sent to peers
-	/// for channels serialized before we included our selected reserve value in the serialized
-	/// data explicitly.
-	pub(crate) fn get_holder_selected_channel_reserve_satoshis(channel_value_satoshis: u64) -> u64 {
+	/// This is used both for outbound and inbound channels and has lower bound of `MIN_THEIR_CHAN_RESERVE_SATOSHIS`
```

Comment:
* Missing article before countable noun is treated as optional and acceptable grammar in most IDE's for code documentation. (default:disabled in intellij)
* Same with period before paragraph end (default:disabled in intellij)
* Same for ',' after e.g or i.e  (default:disabled in intellij)

I am currently using default Intellij grammar for "English(USA)" and it relaxes for these conditions by default. 
I guess it could vary a little based on keyboard selection of user.

We are really discussing these IDE settings in PR's, which i don't think is worth our time in PR.
We can standardize them, if we want.
Either we stick to default acceptable grammar or we can decide to just enable those optional validation checks. 

At the same time, we don't want to discard all grammar and write illegible docs. We should work towards having acceptable grammar and legible docs.

Reply:
> We are really discussing these IDE settings in PR's, which i don't think is worth our time in PR.
> We can standardize them, if we want.
> Either we stick to default acceptable grammar or we can decide to just enable those optional validation checks.

I tend to disagree. There are best practices to write documentation in full, grammatically correct sentences, and we should do our best to follow these. I think whether a particular IDE picks up on some of the details doesn't matter all too much. 

I also think discussing subtleties of wording etc. is time well spent in a PR, since it can make or break a good documentation **and** most importantly can reveal diverging understandings of the subject at hand.

Reply:
Given that the rest of the docs in the codebase is mixed with how strict the we are with period before paragraphs ends
and similar, I can agree on letting it slip :).
Though I also tend to agree with @tnull here. I don't see why we shouldn't make our best efforts with the grammar in new PRs if it's quick to fix and doesn't take much time, given that suggested grammar is _more correct_. IMO when it tends to just be a waste of time and be too nit picky is when time is spent on discussing specific words or sentences that are _equally correct_, and brings no extra clarity.

Reply:
Its not about whether IDE picks it up or not, it has the functionality but the chosen default:disabled means these rules are considered flexible and acceptable. 
I am up for discussing any subtleties, wordings or grammar. But for grammar, saying that only "British English" official rules are "correct" is far from reality. 

And I doubt adding a comma after "e.g." makes or breaks a doc. We are not writing a book here. We need to be practical and subjectively decide how strict we are. Specifically when it adds no additional value in understanding.

Reply:
Mh, I think we're discussing two different things here. Should there be a considerable amount of time spent on splitting hairs regarding individual commas and/or missing articles? No, probably not.

But, I still think reviewers *should* point anything out that they think would improve the quality of the PR, even if it wouldn't warrant having a long discussion about it. Speaking for myself, most of the nits are really meant as suggestions and/or questions, which, at the end of the day, the author may or may not choose to consider.